### DISCUSSION TITLE
Usage instructions should clarify that Windows Terminal (powershell) will not work

### DISCUSSION THREAD

**Discussion by WrayOfSunshine** [TychoTheTaco/Twitch-Drops-Bot#112] [General]
After some testing wherein I thought the instructions for usage were invalid, I realized that my issue was that I was in Windows Terminal (which defaults to Powershell as a terminal type) rather than Command Prompt - this causes issues with the "%cd%" portion of the instructions.

I realize that it says command prompt, but for many less-technical users, it may not be obvious why this is an error if they end up in Terminal instead.

Food for thought, maybe.

**Comment by TychoTheTaco** [comment]
Sure, I could add a separate entry for PowerShell
